File: fontes/c/micro/micro06.c
#include<stdio.h>

int main()
{
 int numero;
 printf("Digite um número de 1 a 5: ");
 scanf("%d", &numero);
 switch(numero)
 {
	 case 1:
	 	printf("Um\n");
	 	break;
	 case 2:
	 	printf("Dois\n");
	 	break;
	 case 3:
	 	printf("Três\n");
	 	break;
	 case 4:
	 	printf("Quatro\n");
	 	break;
	 case 5:
	 	printf("Cinco\n");
	 	break;
	 default:
	 	printf("Número inválido!!!\n");
	 	break;
 }

 return 0;
}

File: fontes/pascal/micro/micro06.pas
program micro05;
var
	numero : integer;
begin
 	write('Digite um numero de 1 a 5: ');
 	readln(numero);
	case numero of
		1 : writeln('Um');
		2 : writeln('Dois');
		3 : writeln('Tres');
		4 : writeln('Quatro');
		5 : writeln('Cinco');
	 	else writeln('Numero invalido!!!');
 	end;
 end.